src/tests/test_mapper.py

```diff
@@ -9,156 +9,119 @@ def open_json_file(json_path):
         return json.load(f)
 
 
-@pytest.fixture()
-def cbs_metadata():
-    return open_json_file("test-data/input-data/cbs-test-metadata.json")
-
-
-@pytest.fixture()
-def cbs_mapping():
-    return open_json_file("test-data/mappings/cbs-mapping.json")
-
-
-@pytest.fixture()
-def cbs_template():
-    return open_json_file(
-        "test-data/template-data/cbs_dataverse_template.json")
-
-
-@pytest.fixture()
-def cbs_result():
-    return open_json_file("test-data/expected-result-data/cbs-result.json")
-
-
-@pytest.fixture()
-def easy_metadata():
-    return open_json_file("test-data/input-data/easy-test-metadata.json")
-
-
-@pytest.fixture()
-def easy_mapping():
-    return open_json_file("test-data/mappings/easy-mapping.json")
-
-
-@pytest.fixture()
-def easy_template():
-    return open_json_file(
-        "test-data/template-data/easy_dataverse_template.json")
-
-
-@pytest.fixture()
-def easy_result():
-    return open_json_file("test-data/expected-result-data/easy-result.json")
-
-
-@pytest.fixture()
-def liss_metadata():
-    return open_json_file("test-data/input-data/liss-test-metadata.json")
-
-
-@pytest.fixture()
-def liss_mapping():
-    return open_json_file("test-data/mappings/liss-mapping.json")
-
-
-@pytest.fixture()
-def liss_template():
-    return open_json_file(
-        "test-data/template-data/liss_dataverse_template.json")
-
-
-@pytest.fixture()
-def liss_result():
-    return open_json_file("test-data/expected-result-data/liss-result.json")
-
+def create_mapper(
```

Comment:
(Minor) Helper functions are conventionally prefixed by `_` to indicate they're private.

Reply:
Done.